tests/test_data.py

```diff
@@ -46,3 +46,42 @@ def test_looks_like_test_file_false(filename):
         from_imports={},
     )
     assert not state.looks_like_test_file()
+
+
+@pytest.mark.parametrize(
+    ("filename"),
+    (
+        "__init__.py",
+        "package/__init__.py",
+        r"package\__init__.py",
+        "project/package/__init__.py",
+        r"project\package\__init__.py",
+    ),
+)
+def looks_like_dunder_init_file_true(filename):
```

Comment:
woops this and the below weren't called `test_`, so they didn't run... 100% test coverage to the rescue though! fixing...

Reply:
Yes, my mistake... 😞  Nice catch coverage ☂️